barely_json/__init__.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
 # encoding: utf-8
 
-'''
```

Comment:
PEP 257 doesn't say anything about single-quotes vs. double-quotes, so using single-quotes for docstrings is perfectly fine as long as they're in triple form (`'''` vs `'`). In `barely_json`, single-quotes are preferred over double-quotes (unless the string contains a single quote).